Library/.rubocop_rules.yml

```diff
@@ -110,13 +118,10 @@ Style/TernaryParentheses:
 
 # dashes in filenames are typical
 Style/FileName:
-  # matches:
-  #   file_name.rb (default)
-  #   file-name.rb, --filename.rb (command names)
-  #   FILENAME.rb (ARGV and ENV)
-  # does not match:
-  #   dashes-and_underscores.rb
-  Regex: !ruby/regexp /^((([\dA-Z]+|[\da-z\+]+)(_([\dA-Z]+|[\da-z\+]+))*|(\-\-)?([\dA-Z]+|[\da-z\+\.]+)([-@]([\dA-Z]+|[\da-z\+\.]+))*))(\.rb)?$/
+  # aa_bb | aa_BB | AABB   (standard): ([\dA-Z]+|[\da-z]+)(\_(([\dA-Z]+|[\da-z]+)))*
+  # aabb | --aa-bb | aa-bb (commands): (\-\-)?[a-z]+(\-[a-z]+)*
+  # aa | aa-bb | aa@b.b    (formulae): ([\da-z\+]+)([\-\.][\da-z\+]+)*(@([\da-z\+]+)([\-\.][\da-z\+]+)*)?
+  Regex: !ruby/regexp /^(([\dA-Z]+|[\da-z]+)(\_(([\dA-Z]+|[\da-z]+)))*|(\-\-)?[a-z]+(\-[a-z]+)*|([\da-z\+]+)([\-\.][\da-z\+]+)*(@([\da-z\+]+)([\-\.][\da-z\+]+)*)?)(\.rb)?$/
```

Comment:
Honestly I think it's probably just worth removing this rule. The message it prints out and the regex itself are both incomprehensible. How about we just allow something much simpler like `[\w@-\+]+.rb`?